src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export function Person({
+  person: { name, age, sex, isMarried, partnerName } = {},
+}) {
+  let msg = '';
+
+  if (!isMarried) {
+    msg = `I am not married`;
+  } else if (sex === 'f') {
+    msg = `${partnerName} is my husband`;
+  } else {
+    msg = `${partnerName} is my wife`;
+  }
```

Comment:
```suggestion
  let maritalStatusMessage = '';

  if (!isMarried) {
    msg = `I am not married`;
  } else if (sex === 'f') {
    msg = `${partnerName} is my husband`;
  } else {
    msg = `${partnerName} is my wife`;
  }
```
Move this logic to the separate function